### PR TITLE
Report source IP on spans

### LINKED ISSUE #121: Report source IP on trace spans

### PR DESCRIPTION
**Pull request by matthewloring**
Fixes #121

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots PTAL.

**Comment by ofrobots** [comment]
1. Can you add a test?
2. How confident are we that the `req.connection` will be a valid object in all three frameworks?

**Comment by matthewloring** [comment]
Added tests to ensure that `req.connection` is valid across the board. @ofrobots ready for another look

File: test/hooks/test-trace-express.js [hunk 1]
```diff
@@ -123,6 +126,23 @@ describe('test-trace-express', function() {
...
+    app.get('/', function (req, res) {
+      res.send(common.serverRes);
+    });
+    server = app.listen(common.serverPort, function() {
+      http.get({port: common.serverPort}, function(res) {
+        var labels = common.getMatchingSpan(expressPredicate).labels;
+        assert.equal(labels[traceLabels.HTTP_RESPONSE_CODE_LABEL_KEY], '200');
+        assert.equal(labels[traceLabels.HTTP_METHOD_LABEL_KEY], 'GET');
+        assert.equal(labels[traceLabels.HTTP_URL_LABEL_KEY], 'http://localhost/');
+        assert.equal(labels[traceLabels.HTTP_SOURCE_IP], '::ffff:127.0.0.1');
```
Comment (ofrobots) [inline test/hooks/test-trace-express.js]: This is dependent on the machine network stack. Maybe check for presence of the property rather than the value.

**Comment by ofrobots** [comment]
LGTM w/ comment.

**Comment by matthewloring** [comment]
@ofrobots I'm going to lock our redis2 fixture version to 2.2.5. It seems like 2.3.0 was just released and it's completely broken. We can take this out once they are stable (or the problem on our end is fixed but we haven't touched that code recently).

**Pull request merged**